sound/soc/sof/pm.c

```diff
@@ -335,8 +335,19 @@ int snd_sof_prepare(struct device *dev)
 		return 0;
 
 #if defined(CONFIG_ACPI)
-	if (acpi_target_system_state() == ACPI_STATE_S0)
+	switch (acpi_target_system_state()) {
+	case ACPI_STATE_S0:
 		sdev->system_suspend_target = SOF_SUSPEND_S0IX;
+		break;
+	case ACPI_STATE_S1:
+	case ACPI_STATE_S2:
+	case ACPI_STATE_S3:
+		sdev->system_suspend_target = SOF_SUSPEND_S3;
+		break;
+	default:
+		sdev->system_suspend_target = SOF_SUSPEND_S3;
```

Comment:
ths is probably not even required as you set to S3 by default outside of the #if right?

Reply:
tools are going to tell me I don't have a default case if I remove it. I could also do a default with just a break, but it's kind of ugly.

Reply:
done